tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+envlist=
+    py27,
+    py34,
+    py35
+
+[flake8]
+max-line-length= 100
+exclude= tests/*
+
+[testenv]
+commands=py.test --tb native {posargs:tests}
+deps =
+    -r{toxinidir}/requirements-dev.txt
+
+[testenv:py27]
+basepython=python2.7
+
+[testenv:py34]
+basepython=python3.4
+
+[testenv:py35]
+basepython=python3.5
```

Comment:
These `basepython` declarations shouldn't be needed anymore for non-ancient versions of Tox.